tests/Containerfile

```diff
@@ -3,7 +3,7 @@ RUN mkdir -p /tmp/rpms
 COPY ./exported-artifacts/ /tmp/rpms/
 RUN dnf install -y --repo local-hirte-snapshot \
     --repofrompath local-hirte-snapshot,file:///tmp/rpms \
-    --nogpgcheck hirte hirte-agent
+    --nogpgcheck hirte hirte-agent hirte-ctl
```

Comment:
Do we actually need `hirtectl` in the e2e tests right now?

Reply:
We don't, but it's a quick test if the package is installable :-)

Reply:
Good point :grin: